### DISCUSSION TITLE
Capital idea!

### DISCUSSION THREAD

**Discussion by w4d3j** [delucca/chia-seeder#2] [General]
I know it's rather early, but from reading the bullets in your Why section I think this has a lot of potential.  I have always wanted to be able to hook external processes to plot phase progression.  

I'd love to help out in any way I can.  I have lots of ideas about what I think it's going to look like, but I'll wait until you've materialized more of your vision before spouting any of my "what-ifs".  Thanks for taking this on bud.

**Comment by delucca** [comment]
Hey, @w4d3j thanks for this post! It is awesome to hear from others that I'm pointing in the right direction 😄 

I've just started coding yesterday! Right now I'm still creating the structure for both the `cli` and `core` modules. This is my last month in the class I'm attending at my master's degree, so I'm going to have way more free time to code from next week and in July.

So, I'm probably going to code most of it in that period 😄 

Regarding contributions, I'm focusing on making them as well documented as I can. Don't know if you have any background in Rust, but it should be pretty straightforward! As soon as I have a testable binary I'm going to ping anyone who is interested!

Thanks again bro 🚀
